.github/workflows/zip_studies.yml

```diff
@@ -0,0 +1,61 @@
+name: Zip studies on push to master branch
+on:
+  # Trigger the workflow on push,
+  # but only for the master branch
+  push:
+    branches:
+      - master
+    paths:
+      - 'public/**'
+jobs:
+  zip_studies:
+    if: github.repository == 'cbioportal/datahub'
+    runs-on: ubuntu-latest
+    timeout-minutes: 1440
+    steps:
+    - id: files
+      uses: jitterbit/get-changed-files@v1
+    - run: | # TODO remove step
```

Comment:
This, and the next few steps should either be removed, or the TODO comment should be removed.